Doc/c-api/exceptions.rst

```diff
@@ -512,7 +512,7 @@ Signal Handling
    and if so, invokes the corresponding signal handler.  If the :mod:`signal`
    module is supported, this can invoke a signal handler written in Python.
 
-   The function attemps to handle all pending signals, and then returns ``0``.
+   The function attempt to handle all pending signals, and then returns ``0``.
```

Comment:
This should be `attempts`

Reply:
@ammaraskar Thank you for the information. 
I fixed it.